architecture/src/androidLibMain/kotlin/navigation/NavigationSpec.kt

```diff
@@ -277,4 +277,34 @@ sealed class NavigationSpec {
             object Normal : Type()
         }
     }
+
+    /**
+     * Opens the a Third Party app if installed on the phone or navigates to the store if it does not exist.
+     * @param packageName The name of the package for which to open the store
+     * @param openMode The [OpenMode] used to determine how to handle wether to open the app or the PlayStore
+     */
+    data class ThirdPartyApp(val packageName: String, val openMode: OpenMode = OpenMode.FALLBACK_TO_STORE): NavigationSpec() {
```

Comment:
I think you and @corrado4eyes made the same thing.

Reply:
AFAIK also for iOS though.

Reply:
Is there a PR for that @corrado4eyes ?

Reply:
I guess Tijl is referring to the enhance I've made for navigation for NavigationSpec.Browser. But that is in order to open the browser

Reply:
Ok. Im making some changes to support it on iOS as well.

Reply:
@thoutbeckers @avdyushin @corrado4eyes Made a proper 3rd party linker in iOS as well, though i wasnt able to properly test it since demo seems to be broken when running on Arm and simulator doesnt support the AppStore

Reply:
This functionality (open browser) is in the old x.bike app, so it needs to be in the new.

If this task was skipped then it's a lucky break.

The issue is here: https://github.com/splendo/kaluga/issues/205

Reply:
Open Browser was already supported. You could technically already use it to open the App/Play Store. But this implementation adds an explicit route for 3rd party apps.

In addition, for Android this implementation checks whether the store itself is actually available.
On iOS, instead of using the browser we use the actual storeKit feature for it, which also supports tracking ids.

Reply:
Yeah I meant open browser in case app is not installed.

Not sure what @corrado4eyes used on iOS but he approved the PR in slack so I assume it will work for him.